tests/sync_pipfile_test.py

```diff
@@ -54,3 +54,27 @@ def test_sync_dev_pipfile_no_original(tmp_path):
         text = setup_file.read_text()
         assert "pytest~=5.1" in text, text
         assert "requirementslib~=1.5" in text, text
+
+
+@pytest.mark.parametrize(
+    ("source_pipfile_dirname", "update_count"),
+    [("dash_or_underscore_0", 23)],
+)
+def test_sync_underscore_or_dash(
+    capsys, tmp_path, shared_datadir, source_pipfile_dirname, update_count
+):
```

Comment:
After [this suggestion](https://github.com/Madoshakalaka/pipenv-setup/pull/86/files#r752446810) and the one above:

```suggestion
def test_sync_underscore_or_dash(shared_datadir):  # type: pathlib.Path
```